packages/fxa-content-server/app/styles/_breakpoints.scss

```diff
@@ -12,12 +11,52 @@ $breakpoints: (
 );
 
 @mixin respond-to($breakpoint) {
-  @if map-has-key($breakpoints, $breakpoint) {
-    @media #{map-get($breakpoints, $breakpoint)} {
+  @include breakpoint-error-check($breakpoint, $media-queries) {
+    @media #{map-get($media-queries, $breakpoint)} {
       @content;
     }
+  }
+}
+
+// NOTE: We will replace the `media-queries` map and `respond-to` mixin
+// with the `breakpoints` map, `min-width` mixin, and `max-width`
+// mixin. Once this is complete we can remove the former pair.
+// TODO: add example outputs to our documentation
```

Comment:
Would you mind creating a tech debt issue to make sure we clean up the old syntax?

Reply:
Will do, I think it probably belongs under the Settings Redesign epic so I'll add it there today. 🙂